src/app/app.component.ts

```diff
@@ -6,5 +6,4 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'OPTC Pirate Rumble Database';
```

Comment:
What's the reason for the title removal?

Reply:
it was not being used 😄